cmd/commandfuncs.go

```diff
@@ -74,6 +74,10 @@ func cmdStart(fl Flags) (int, error) {
 	// sure by giving it some random bytes and having it echo
 	// them back to us)
 	cmd := exec.Command(os.Args[0], "run", "--pingback", ln.Addr().String())
```

Comment:
I don't have access to Windows machine with the same issue, but I wondered if using `os.Executable` would fix it. The error message complains about relative path, but `os.Executable` returns the full path. What do you think?

Reply:
`os.Executable` is fine. It's used in other places where caddy binary path is needed, i.e. experimental `upgrade` command.

Reply:
So is this error ignorable without impacting the `Start` later?

Reply:
It's in the [official examples](https://pkg.go.dev/os/exec):

```
cmd := exec.Command("prog")
if errors.Is(cmd.Err, exec.ErrDot) {
	cmd.Err = nil
}
if err := cmd.Run(); err != nil {
	log.Fatal(err)
}
```